frontend/src/index.html

```diff
@@ -7,6 +7,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
   <link rel="icon" type="image/x-icon" href="favicon.ico?v1">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.12/css/all.css" integrity="sha384-G0fIWCsCzJIMAVNQPfjH08cyYaUtMwjJwqiRKxxE/rx96Uroj1BtIQ6MLJuheaO9" crossorigin="anonymous">
```

Comment:
I see that we are already having a 'volontulo-icon' component for material-icons. From my perspective, on one hand we shouldn't mix font-awesome with material-icons, on the other hand I think that having a component for creating icons is a little bit of overkill as those could be written by hand, probably in pure html, instead of managing components in JS.
Unless we have a really dynamic use case for that, where icons could change in the same markup due to some data refresh.

Reply:
@magul we agreed to install package you suggests and I'll totally do that, but first I would love to close my current PR and 404 page. I believe it can stay for now, I'll remove it after installing package.